core-services/prow/02_config/openshift/cluster-monitoring-operator/_pluginconfig.yaml

```
@@ -10,6 +10,10 @@ label:
       - arajkumar
       - simonpasquier
       label: backport-risk-assessed
+      allowed_users:
```

shouldn't it be a new item under the openshift/cluster-monitoring-operator key?

```suggestion
    - allowed_users:
```

---

like in https://github.com/openshift/release/blob/6f5b16a432d9628939f7a96e3efe32e4d0a66e69/core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml#L7-L16